tests/img_test/config.in

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+TEST_NAME="img_test"
+
+export EMAILS=""
+export MAIL_SCRIPT="$RUN_DIR/src/disrupt.sh"
+export FACTS_SOURCE="$TEST_DIR/$TEST_NAME/facts.lf"
```

Comment:
Maybe we can pull the variables that are unlikely to change outside the test_name variable from another file so we don't have to change them everywhere if we ever do - like source a base_config.in file? Then you can rewrite what you care about.

Reply:
Ah good call.